src/addons/transitions/ReactCSSTransitionGroupChild.js

```diff
@@ -95,7 +96,7 @@ var ReactCSSTransitionGroupChild = React.createClass({
   flushClassNameQueue: function() {
     if (this.isMounted()) {
       this.classNameQueue.forEach(
-        CSSCore.addClass.bind(CSSCore, this.getDOMNode())
+        CSSCore.addClass.bind(CSSCore, findDOMNode(this))
```

Comment:
no, other than that I opened this file twice and updated one call at a time and honestly didn't realize that it was the same file :)